index.bs

```diff
@@ -4149,7 +4162,11 @@ To <dfn>trigger aggregatable attribution</dfn> given an [=attribution trigger=]
 1. If the result of running [=check if attribution should be blocked by rate limits=]
     with |trigger|, |sourceToAttribute|, and |rateLimitRecord| is not null,
     return it.
-1. If |sourceToAttribute|'s [=attribution source/number of aggregatable attribution reports=] value is equal to [=max aggregatable reports per source=][0], then:
+1. If |sourceToAttribute|'s [=attribution source/max number of aggregatable reports=] is greater than
+    [=default max aggregatable reports per source=] and |trigger|'s [=attribution trigger/trigger context ID=] is null:
+    1. Return the [=triggering result=] ("<code>[=triggering status/dropped=]</code>",
+        ("<code>[=trigger debug data type/trigger-aggregate-mismatched-max-aggregatable-reports-config=]</code>", null)).
+1. If |sourceToAttribute|'s [=attribution source/number of aggregatable attribution reports=] value is equal to its [=attribution source/max number of aggregatable reports=], then:
```

Comment:
If we are including the source debug key, I guess that would be OK, but isn't the limit on its own relatively high entropy since it's between 0 and 100?

Reply:
The text, `, then:`, is used in a few `if` statements in the spec. Why do you propose to remove it here, specifically?

Reply:
We are including source_event_id, so it’s ok to use the source’s max_aggregatable_reports. You can add back “, then”.

Reply:
Amended the limit in the debug report.